<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QUaModbusValueWidgetEdit</class>
 <widget class="QWidget" name="QUaModbusValueWidgetEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>211</width>
    <height>149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QFrame" name="frameSettings">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="labelId">
        <property name="text">
         <string>Value Id :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditId"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelType">
        <property name="text">
         <string>Value Type :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QUaModbusReadOnlyComboBox" name="comboBoxType"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelOffset">
        <property name="text">
         <string>Offset :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="spinBoxOffset">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameCyclic">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0" colspan="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelCyclicMode">
        <property name="text">
         <string>Cyclic Mode : </string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QUaModbusReadOnlyComboBox" name="comboBoxCyclicMode"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelCyclicPeriod">
        <property name="text">
         <string>Cyclic Period : </string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="spinBoxCyclicPeriod">
        <property name="maximum">
         <number>999999999</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QUaModbusReadOnlyComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">QUaModbusCommonWidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
